CMakeLists.txt

```diff
@@ -102,6 +111,8 @@ include_directories(include)
 # Location of MPI
 include_directories(${MPI_DIR}/include)
 link_directories(${MPI_DIR}/lib)
+message(STATUS "MPI include dir: ${MPI_DIR}/include")
+message(STATUS "MPI lib dir: ${MPI_DIR}/lib")
```

Comment:
What if MPI uses `lib64`?

Reply:
That is a good point. It was originally stated like this. 
I can add a test to check whether or not the lib64 folder exists, and if not, back to lib